### ISSUE TITLE
build issue with jaxb2-maven-plugin 1.6 and java-8

### ISSUE THREAD

**Issue by metas-ts** [metasfresh/metasfresh#492]
The error message is:
```
Failed to read schema document 'xjc.xsd', because 'file' access is not allowed due to restriction set by the accessExternalSchema property.
```

it was no problems with jaxb2-maven-plugin version 2.2 but we recently switched back to 1.6. because of mojohaus/jaxb2-maven-plugin#24

**Comment by metas-dh** [comment]
checked with tobi about way to IT: if the issue wasn't solved, the printing client  (normal printing client) wouldn't work:
* checked printing of orders (normal printing, not print preview): working fine, no errors
* printing of jaspers worked fine as well

**Comment by sergiomichels** [comment]
When this will be released?

**Comment by metas-ts** [comment]
Good Morning Sérgio,
and sorry for the late reply.
this fix was released last week, with metasfresh v4.40.39.
By the way, here are the fully release notes: http://metasfresh.com/2016/10/21/release-4-40-39/

Best regards,
Tobias

**Comment by metas-mk** [comment]
Was in Release 4.40.39 (2016-42) on Friday last week.
https://github.com/metasfresh/metasfresh/blob/master/ReleaseNotes.md

Greetings
Mark


Von: Sérgio Michels [mailto:notifications@github.com]
Gesendet: Mittwoch, 26. Oktober 2016 17:59
An: metasfresh/metasfresh
Betreff: Re: [metasfresh/metasfresh] build issue with jaxb2-maven-plugin 1.6 and java-8 (#492)


When this will be released?
